client/maisie/interface.py

```diff
@@ -10,15 +10,11 @@
 
 
 @click.group()
-@click.option(
-    "--output_json", is_flag=True, help="Display output in pure json instead of a table"
-)
+@click.option("--output", help="Specify format of the output as e.g. json")
```

Comment:
Maybe we should add a default value for this parameter, for example `--output=default` or `--output=table`?